### ISSUE TITLE
RBF_Final.pth第一个batchnorm的 mean & val 为什么这么大

### ISSUE THREAD

**Issue by WendShadow** [biubug6/Face-Detector-1MB-with-landmark#26]
合并batchnorm后发现第一个卷积层的权重过小
对比原版的Ultra-Light-Fast-1M的原始model， RBF_Final.pth的第一个batchnorm的mean与val参数大的有些异常，求解。。。
RBF_Final.pth  first batchnorm :
![RBF_Final_1thbatchnorm](https://user-images.githubusercontent.com/16531210/75736977-a5899e00-5d39-11ea-98ff-9d9c1fcbea03.png)
Ultra-Light-Fast-Generic-Face-Detector-1MB/models/pretrained/version-RFB-320.pth first batchnorm : 
![version-RFB-320_1thbatchnorm](https://user-images.githubusercontent.com/16531210/75737094-ec779380-5d39-11ea-9338-84225fedbb94.png)
